frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-details.tsx

```diff
@@ -6,19 +6,52 @@ import {
   SectionHeading,
   ResourceSummary,
   Kebab,
+  DetailsItem,
 } from '@console/internal/components/utils';
 import { CatalogSourceListPage, CatalogSourceListPageProps } from '../catalog-source';
 import { OperatorHubKind } from '.';
 import { OperatorHubModel } from '../../models';
+import { editDefaultSourcesModal } from '../modals/edit-default-sources-modal';
 
-const OperatorHubDetails: React.FC<OperatorHubDetailsProps> = ({ obj }) => {
+const OperatorHubDetails: React.FC<OperatorHubDetailsProps> = ({ obj: operatorHub }) => {
   const { t } = useTranslation();
   return (
     <div className="co-m-pane__body">
       <SectionHeading
         text={t('operator-hub-details~{{resource}} details', { resource: OperatorHubModel.label })}
       />
-      <ResourceSummary resource={obj} podSelector="spec.podSelector" showNodeSelector={false} />
+      <div className="row">
+        <div className="col-sm-6 col-xs-12">
+          <ResourceSummary
+            resource={operatorHub}
+            podSelector="spec.podSelector"
+            showNodeSelector={false}
+          />
+        </div>
+        <div className="col-sm-6 col-xs-12">
+          <div className="co-m-pane__details">
+            <DetailsItem
+              label="Default Sources"
+              obj={operatorHub}
+              path="status.sources"
+              canEdit
```

Comment:
@spadgett Do we need to do an access check here to make sure the user can edit the OperatorHub resource?

Reply:
Good catch! Yes, let's add an access review.

Reply:
Added the check